pkg/resolution/resolver/bundle/resolver.go

```diff
@@ -112,6 +117,11 @@ func (r *Resolver) Resolve(ctx context.Context, params []pipelinev1.Param) (fram
 	return GetEntry(ctx, kc, opts)
 }
 
+// ResolveUrl uses the given url to resolve the requested file or resource.
+func (r *Resolver) ResolveUrl(ctx context.Context, url string) (framework.ResolvedResource, error) {
+	return nil, nil
```

Comment:
Should this return an unimplemented error instead?

Reply:
Ah! good point. I can do that.

Reply:
fixed.